schematic/store/synapse.py

```diff
@@ -615,7 +630,14 @@ def _get_file_entityIds(self, dataset_files: List,  only_new_files: bool = False
             
             # find new files (that are not in the current manifest) if any
             for file_id, file_name in dataset_files:
-                if not file_id in manifest["entityId"].values:
+                if "entityId" in manifest.columns:
```

Comment:
The original code here assumes that when filling in "Filename" column and "entityId" column, those two columns are already present in the existing manifest. But when `use_annotations` is set to True and there is no existing manifest, we still want File name and "entityId" column to be filled. See comments from line 637-640